spec/how_is_spec.rb

```diff
@@ -109,21 +109,21 @@
   end
 
   # Disable 'cop' that is violated by every .generate_frontmatter() calls.
```

Comment:
while you're working on this file, could you change "calls" to "call", here? also I guess replace `'cop'` with `rule` or something.